waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/DidCreation.kt

```diff
@@ -49,9 +50,26 @@ object DidCreation {
                 }
             }
         }) {
-            getWalletService().createDid(
-                DidWebMethodName, extractDidCreateParameters(DidWebMethodName, context.request.queryParameters)
-            ).let { context.respond(it) }
+            val parameters = extractDidCreateParameters(DidWebMethodName, context.request.queryParameters)
+            runCatching {
```

Comment:
I think this is not required anymore (catching the exception in controller), since there is a type - ConflictException - for it in service-commons WebExceptions that is thrown by the did-service, so status-pages should be able to display it